index.js

```diff
@@ -58,12 +43,6 @@ Geocoder.prototype = inherit(mapboxgl.Control, {
   onAdd: function(map) {
     this.request = new XMLHttpRequest();
 
-    this.container = this.options.container ?
-      typeof this.options.container === 'string' ?
-      document.getElementById(this.options.container) :
-      this.options.container :
-      map.getContainer();
```

Comment:
It might be important, but as of right now it's impossible. You'll need to lobby for that as an API feature on the GL JS side, while this PR will fix the arguably more important issue of this control being broken.